### DISCUSSION TITLE
I don't understand the sessions and meeting keys.

### DISCUSSION THREAD

**Discussion by MrMipya** [br-g/openf1#75] [Q&A] [answered]
What are session and meeting keys, are they random or specific or based on a seed can someone please explain.

**Comment by mohamedhadrami** [comment, marked as answer]
It's what F1 uses for live timing. It's just fed into the API. I think F1 has a system but I can't seem to figure it out. Sometimes it's incremental and sometimes there's a massive jump. It would be nice to be able to predict future meeting and session keys, if anyone can decipher it.
